reconcile/cli.py

```diff
@@ -1972,11 +1974,16 @@ def ocm_upgrade_scheduler(ctx):
 @environ(["APP_INTERFACE_STATE_BUCKET", "APP_INTERFACE_STATE_BUCKET_ACCOUNT"])
 @click.pass_context
 def ocm_upgrade_scheduler_org(ctx):
-    import reconcile.aus.ocm_upgrade_scheduler_org
+    from reconcile.aus.base import AdvancedUpgradeSchedulerBaseIntegrationParams
+    from reconcile.aus.ocm_upgrade_scheduler_org import (
+        OCMClusterUpgradeSchedulerOrgIntegration,
+    )
 
     run_class_integration(
-        integration=reconcile.aus.ocm_upgrade_scheduler_org.OCMClusterUpgradeSchedulerOrgIntegration(
-            NoParams()
+        integration=OCMClusterUpgradeSchedulerOrgIntegration(
+            AdvancedUpgradeSchedulerBaseIntegrationParams(
+                # pass in env or org here for sharding
```

Comment:
what does it mean? we can later add some `--organization` or `--ocm-environment` and filter base on this ?

Reply:
yes exactly